app/helpers/homepage_helper.rb

```diff
@@ -0,0 +1,19 @@
+module HomepageHelper
+  def organisations_json
+    GdsApi.content_store.content_item("/government/organisations")
```

Comment:
It's probably worth considering how this could fail and handling it. We have exception handling for loading the homepage item, we'd not want this to break the homepage.

It does seem very strange to use a different content item to load the data for this page. I'd definitely think about if it can be done with a different approach. If we need to use a content item link expansion seems the right way rather than pulling straight from an unrelated URL (this /government/organisations path seems rather whacky since it doesn't even have a page on GOV.UK https://www.gov.uk/government/organisations 🤔 )

Reply:
By the way - for reference this is a link the to the underlying architectural principle: https://docs.google.com/document/d/1Oft4akc6dZfhhOjosNPbFpcLUOUjz7YG7QPcVZi8hww/edit#heading=h.h8e5mqu62b4b

Reply:
@kevindew you and @thomasleese have convinced me the content item is not the right thing for this.
I'll keep this open but give it a bit more thought and try a few strategies with these tests.

Thanks very much for the review!